jdk/make/closed/autoconf/custom-spec.gmk.in

```diff
@@ -134,4 +134,9 @@ else
   OPENJ9_VM_BUILD_DIR = $(OUTPUT_ROOT)/vm
 endif
 
+# Enable use of j9cfg.h in openjdk native code.
+$(foreach var, \
+	CFLAGS_JDKEXE CFLAGS_JDKLIB CXXFLAGS_JDKEXE CXXFLAGS_JDKLIB, \
+	$(eval $(var) += -I$(SUPPORT_OUTPUTDIR)/openj9_include))
```

Comment:
`SUPPORT_OUTPUTDIR` does not exist in jdk8: this and `OpenJ9.gmk` will need changes.

Reply:
Changed it to `JDK_OUTPUTDIR`.